executor/executor_test.go

```diff
@@ -6683,7 +6683,7 @@ func (s *testSerialSuite) TestCoprocessorOOMAction(c *C) {
 		c.Check(err, IsNil)
 		tk.Se = se
 		tk.MustExec("use test")
-		tk.MustExec("set @@tidb_distsql_scan_concurrency = 2")
+		tk.MustExec("set @@tidb_distsql_scan_concurrency = 4")
```

Comment:
modify `tidb_distsql_scan_concurrency` to 4 in order to prevent non-keep order case from being oom.